doc/source/whatsnew/v0.20.0.txt

```diff
@@ -50,6 +50,25 @@ fixed-width text files, and :func:`read_excel` for parsing Excel files.
    pd.read_fwf(StringIO(data)).dtypes
    pd.read_fwf(StringIO(data), dtype={'a':'float64', 'b':'object'}).dtypes
 
+
+.. _whatsnew_0200.enhancements.errors:
+
+pandas errors
+^^^^^^^^^^^^^
+
+We are adding a standard public location for all pandas exceptions & warnings ``pandas.errors``. (:issue:`14800`). Previously
+these exceptions & warnings could be imported from ``pandas.core.common`` or ``pandas.io.common``. These exceptions and warnings
+will be removed from the ``*.common`` locations in a future release. (:issue:`15541`)
+
+The following are now part of this API:
+
+.. ipython:: python
+
+   import pprint
+   from pandas import errors
+   excs = [ e for e in dir(errors) if not e.startswith('_') ]
+   pprint.pprint(excs)
```

Comment:
This is a bit of a nit, but I would rather just list the error type explicitly here. That ensures this list remains constant even if/when we add more errors and rebuild the docs later.